src/components/Logo.js

```diff
@@ -48,8 +24,8 @@ const Logo = ({ isWithGang, size = 'default' }) => {
 };
 
 Logo.propTypes = {
-  isWithGang: PropTypes.bool,
   size: PropTypes.string,
+  type: PropTypes.string,
```

Comment:
I guess this property does not make sense anymore as it is not used within the component.

Reply:
corrected